content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -268,6 +268,24 @@ The annotation `kubernetes.io/limit-ranger` records that resource defaults were
 and they were applied successfully.
 For more details, read about [LimitRanges](/docs/concepts/policy/limit-range).
 
+### addonmanager.kubernetes.io/mode
+
+Example: `addonmanager.kubernetes.io/mode: "Reconcile"`
+
+Used on: All objects
+
+To specify how an add-on should be managed, you can use the `addonmanager.kubernetes.io/mode` label.
+This label can have one of three values: `Reconcile`, `EnsureExists`, or `Ignore`.
+
+- `Reconcile`: Addon resources will be periodically reconciled with the expected state. If there are any differences,
+the add-on manager will recreate, reconfigure or delete the resources as needed. This is the default mode if no label is specified.
+- `EnsureExists`: Addon resources will be checked for existence only but will not be modified after creation.
+The add-on manager will create or re-create the resources when there is no instance of the resource with that name.
+- `Ignore`: Addon resources will be ignored. This mode is useful for add-ons that are not compatible with
+the add-on manager or that are managed by another controller.
```

Comment:
/lgtm

I'd suggest we wrap and indent the list items in a different way.

```suggestion
- `Reconcile`: Addon resources will be periodically reconciled with the expected state.
  If there are any differences, the add-on manager will recreate, reconfigure or delete
  the resources as needed. This is the default mode if no label is specified.
- `EnsureExists`: Addon resources will be checked for existence only but will not be
  modified after creation. The add-on manager will create or re-create the resources
  when there is no instance of the resource with that name.
- `Ignore`: Addon resources will be ignored. This mode is useful for add-ons that are
  not compatible with the add-on manager or that are managed by another controller.
```

Reply:
Hi @tengqm, I agree with you on the line wrapping. This document, in particular is 1153 lines long, I see literally hundreds of lines that are too long. I don't mind wrapping them as well. Do you want me to just do that?

Reply:
The LGTM I left means that I'm okay with this PR merged as is.
The line wrapping is a nit to be fixed in case you have another push.
As for other lines in the same file, maybe a dedicated tidy up PR is needed.

Reply:
Got it. Thanks